apps/barrel_store/test/barrel_metrics_SUITE.erl

```diff
@@ -32,7 +32,7 @@
 -export([init/3, increment/3]).
 
 all() -> [ plugin
-         , measures
+         %% , measures
```

Comment:
@BernardNotarianni imo the commit should only be related to the ticket. It makes the historic easier to read later. It's also easier to edit it later in context. So if you don't mind to remove it, let's do it :)